css/modules/button.css

```diff
@@ -0,0 +1,13 @@
+.prev-btn{
```

Comment:
let the main class be btn, btn--next is extension, btn--previous is an extension